lib/Doctrine/ORM/Cache/Region/DefaultRegion.php

```diff
@@ -100,30 +102,28 @@ public function get(CacheKey $key)
      */
     public function getMultiple(CollectionCacheEntry $collection)
     {
-        $keysToRetrieve = array();
-
-        foreach ($collection->identifiers as $index => $key) {
-            $keysToRetrieve[$index] = $this->name . '_' . $key->hash;
-        }
-
-        $items = array_filter(
-            array_map([$this->cache, 'fetch'], $keysToRetrieve),
-            function ($retrieved) {
-                return false !== $retrieved;
+        $result = array();
```

Comment:
@holtkamp it's not a convention from Doctrine, but we should make it.
There is clear advantage of applying early returns; you increase code readability, reduce the cyclomatic complexity of your algorithm and also have fewer bytecode instructions to follow.